### DISCUSSION TITLE
10 star ratings

### DISCUSSION THREAD

**Discussion by isle9** [navidrome/navidrome#2565] [Ideas]
Any chance we get a 10 star rating option in the future?

**Comment by somedevreally** [comment]
this would be great, specially for smart playlist. 
 
 right now am using this like
1 star never play or dont like
2 star explicit lyrics
3 star song of the week
and so on.
maybe its dumb way, but it works for me :)

**Reply by isle9** [reply]
Yeah, I use the ratings like that too since they are the only way to tag stuff unfortunately. It would be great if we there was support for user tags in some way.
